soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -106,6 +111,18 @@ void EnCow_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnCow* this = (EnCow*)thisx;
     s32 pad;
 
+    // Move left cow in lon lon tower
+    if (EnCow_IsSecondCowInTower(thisx, globalCtx)) {
+        this->actor.world.pos.x = -229.0f;
+        this->actor.world.pos.z = 157.0f;
+        this->actor.shape.rot.y = 15783.0f;
+    }
```

Comment:
are these cows moved in 3drando and/or 64rando?

Reply:
Not sure about 3drando but yes for 64rando